packages/create/bin/create-stacks-workspace.ts

```diff
@@ -155,14 +155,26 @@ async function getConfiguration(argv: yargs.Arguments<CreateStacksArguments>) {
 
 async function main(parsedArgv: yargs.Arguments<CreateStacksArguments>) {
     const { nxVersion, dir, overwrite, ...forwardArgv } = parsedArgv;
-    const { name } = forwardArgv;
+    const { name, skipGit } = forwardArgv;
 
     const argumentsToForward = unparse(
         normaliseForwardedArgv(forwardArgv) as unparse.Arguments,
         { alias: { packageManager: ['pm'], interactive: ['i'] } },
     );
 
-    const root = path.resolve(dir);
+    let root = path.resolve(dir);
+
+    // Stacks-cli gives us the working directory + project name via --dir
+    // We need to strip out the project name as create-nx-workspace will create it
+    if (overwrite) {
+        const directoryName = path.basename(dir);
+        if (paramCase(directoryName) === name) {
+            if (fs.existsSync(root)) {
+                fs.rmSync(root, { recursive: true, force: true });
+            }
+            root = path.dirname(root);
+        }
+    }
```

Comment:
I feel like this logic could do with a unit test with different cases, as it's a bit unclear what is happening here.

Given:
- cwd `/root`
- `npx @ensono-stacks/create-nx-workspace ProjectName --dir /root/ProjectName --overwrite`

Then:
- `root = '/root/ProjectName`
- `directoryName = `ProjectName`
- the if condition is true
- `rm -rf /root/ProjectName`
- `root = '/root'`

So it all checks out.

My only question is that `npx create-nx-workspace` will paramCase the project name. And create `/root/project-name`, while Stacks CLI behavior is `/root/ProjectName`. I wonder if this may be unexpected behavior from Stacks CLI perspective?

But having checked `create-nx-workspace`, there's no option to pass a custom dir there :/

Reply:
I've given it another stab.

`--dir=./path/to/project` - will compile the project to that folder as long as it doesnt exist
`--dir=./path/to/project --overwrite` - will compile the project to that folder and replace the folder if it already exists.

If no dir is supplied it will write to the cwd with the param case project name

Reply:
Unfortunately Node doesn't support renaming folders that are not empty, so have to rely on copy (which is what we wanted to avoid). So using --dir is significantly longer.